plugin/plugin.go

```diff
@@ -129,6 +129,14 @@ func (p *plugin) isSupportedInt(field *descriptor.FieldDescriptorProto) bool {
 	return false
 }
 
+func (p *plugin) isSupportedDouble(field *descriptor.FieldDescriptorProto) bool {
+	switch *(field.Type) {
+	case descriptor.FieldDescriptorProto_TYPE_DOUBLE:
```

Comment:
I think the golang codegen will work for floats, maybe worth adding? `FieldDescriptorProto_TYPE_FLOAT` here?